### PR TITLE
Change read markers to use CSS transitions

### PR DESCRIPTION
**Pull request by dbkr**
Removes one of the two places we use Velocity, so we're one step
closer to getting rid of it for good.

Should therefore fix the fact that Velocity is leaking data entries
and therefore `<hr>` elements.

Hopefully also makes the logic in getEventTiles incrementally simpler,
if still somwewhat byzantine.

Requires https://github.com/vector-im/riot-web/pull/11521

### CODE REVIEW

**Comment by turt2live** [comment]
Fixes https://github.com/vector-im/riot-web/issues/11496 ?

**Comment by dbkr** [comment]
> Fixes [vector-im/riot-web#11496](https://github.com/vector-im/riot-web/issues/11496) ?

Sadly not - read markers == the green lines, read receipts == the little faces

Yeah, they are quite edge-casey, especially around the MELS stuff: adding some tests made me a bit more confident about doing this.

File: test/components/structures/MessagePanel-test.js [hunk 1]
```diff
@@ -191,50 +263,4 @@ describe('MessagePanel', function() {
             }, 100);
         }, 100);
     });
-
-    it('shows only one ghost when the RM moves twice', function() {
```
Comment (dbkr) [inline test/components/structures/MessagePanel-test.js]: I should say: this test was really testing that the animations didn't break (https://github.com/matrix-org/matrix-react-sdk/pull/254). The animations will now work fine however many ghosts we end up showing so this test is no longer useful as it shows multiple ghosts by design.
